.github/workflows/test.yml

```diff
@@ -90,7 +90,7 @@ jobs:
             - name: Test KFP libs (shared and v1) and run a workflow
               timeout-minutes: 120
               run: |
-                  source kind/requirements.env
+                  source scripts/k8s-setup/requirements.env
```

Comment:
should we define `scripts/k8s-setup` as a variable?

Reply:
Fixed. Thanks